src/Optimizer/Configuration/AutoExtensionsConfiguration.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace AmpProject\Optimizer\Configuration;
+
+use AmpProject\Amp;
+use AmpProject\Exception\InvalidExtension;
+use AmpProject\Extension;
+use AmpProject\Format;
+use AmpProject\Optimizer\Exception\InvalidConfigurationValue;
+use ReflectionClass;
+
+/**
+ * Configuration for the AutoExtensions transformer.
+ *
+ * @property string $format                  Specifies the AMP format. Defaults to `AMP`.
+ * @property bool   $autoExtensionImport     Set to `false` to disable the auto extension import. Defaults to `true`.
+ * @property bool   $experimentBindAttribute Enables experimental conversion of bind attributes. Defaults to `false`.
+ *
+ * @package ampproject/amp-toolbox
+ */
+final class AutoExtensionsConfiguration extends BaseTransformerConfiguration
+{
+
+    /**
+     * Configuration key that specifies the AMP format.
+     *
+     * @var string
+     */
+    const FORMAT = 'format';
+
+    /**
+     * Configuration key that can disable the automatic importing of extension.
+     *
+     * @var bool.
+     */
+    const AUTO_EXTENSION_IMPORT = 'autoExtensionImport';
+
+    /**
+     * Configuration key that enables experimental conversion of bind attributes.
+     *
+     * @var bool
```

Comment:
The `@var` here defines the type of the constant, not the type of the value that this configuration key refers to.
```suggestion
     * @var string
```